kraken/node_actions/common_node_functions.py

```diff
@@ -83,10 +137,21 @@ def check_service_status(node, service, ssh_private_key, timeout):
     for service_name in service:
         logging.info("Checking status of Service: %s" % (service_name))
         stdin, stdout, stderr = ssh.exec_command(
-            "systemctl status %s  | grep '^   Active' " "|  awk '{print $2}'" % (service_name)
+            "systemctl status %s  | grep '^   Active' "
+            "|  awk '{print $2}'" % (service_name)
         )
         service_status = stdout.readlines()[0]
-        logging.info("Status of service %s is %s \n" % (service_name, service_status.strip()))
+        logging.info(
+            "Status of service %s is %s \n" % (service_name, service_status.strip())
+        )
         if service_status.strip() != "active":
-            logging.error("Service %s is in %s state" % (service_name, service_status.strip()))
+            logging.error(
+                "Service %s is in %s state" % (service_name, service_status.strip())
+            )
     ssh.close()
+
+
+# Get the resource version for the specified node
+def get_node_resource_version(node):
+    v1 = client.CoreV1Api()
+    return v1.read_node(name=node).metadata.resource_version
```

Comment:
Let's add this call to https://github.com/chaos-kubox/krkn/blob/main/kraken/kubernetes/client.py (the corev1api is already set up there) and make a common function just to read node and then get the needed resource version so we could reuse the read_node to do a variety of calls